opentelemetry-sdk/CHANGELOG.md

```diff
@@ -282,6 +282,10 @@ limit.
   now:
   `opentelemetry_sdk::logs::{ExportResult, LogBatch, LogExporter};`
 
+- *Breaking* `opentelemetry_sdk::LogRecord::default()` method is removed.
```

Comment:
- *Breaking* `opentelemetry_sdk::LogRecord::default()` method is removed.
   Use `Logger::create_log_record()` method instead.